common.sh

```diff
@@ -17,6 +17,8 @@ AFL_SRC=$(dirname $(dirname $SCRIPT_DIR))/AFL
 FUZZ_CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address -fsanitize-coverage=trace-pc-guard,trace-cmp,trace-gep,trace-div"
 CORPUS=CORPUS-$EXECUTABLE_NAME_BASE
 JOBS=8
+# Every "gcloud compute ssh/scp/etc" needs --zone, otherwise throws warning
+GCLOUD_ZONE="us-west1-b"
```

Comment:
The benchmark scripts don't use it, do they?  Maybe an engine_comparison/common.sh file would be a better alternative.